examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -270,7 +287,7 @@ following command:
 To build the example with release configuration that disables the diagnostic
 features like logs and command-line interface, run the following command:
 
-    $ west build -b build-target -- -DCONF_FILE=prj_release.conf
+    $ west build -b build-target --sysbuild -- -DCONF_FILE=prj_release.conf
```

Comment:
And what about using `FILE_SUFFIX` instead of `CONF_FILE`?

Reply:
Good catch! Thanks!